templates/rack_timeout.rb

```diff
@@ -1 +1,3 @@
-Rack::Timeout.timeout = (ENV["RACK_TIMEOUT"] || 10).to_i
+if Rails.env.production? || Rails.env.staging?
```

Comment:
:+1: to this comment. @jakecraige can you please add that change, and rebase to latest master?